2_Variables_and_Operators.ipynb

```diff
@@ -510,14 +472,40 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Convert a float and an integer to a string with the str() statement \n",
+    "# Convert a float to a string with the str() function \n",
     "myFloat = 4.5\n",
     "myFloatString = str(myFloat)\n",
+    "str(myFloat)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "float(myFloatString)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Convert an integer to a string with the str() function\n",
     "myInteger = 5\n",
     "myIntegerString = str(myInteger)\n",
-    "print(f\"My strings are {myFloatString} and {myIntegerString}\")\n",
-    "print(f\"My string converted to integer is {int(myIntegerString)}\")\n",
-    "print(f\"My string converted to float is {float(myFloatString)}\")"
```

Comment:
Also remove text cell underneath it: 
_in the example above you ... with the `f´ method._

Reply:
Done :+1: